test/test_functional_tensor.py

```diff
@@ -692,6 +433,250 @@ def test_scale_channel(self):
+        )
+        out_pil_tensor = torch.from_numpy(np.array(out_pil_img).transpose((2, 0, 1)))
+
+        if scripted_affine:
+            fn = torch.jit.script(F.affine)
+        else:
+            fn = F.affine
+
+        out_tensor = fn(
+            tensor, angle=angle, translate=[0, 0], scale=1.0, shear=[0.0, 0.0], interpolation=NEAREST
+        ).cpu()
+
+        if out_tensor.dtype != torch.uint8:
+            out_tensor = out_tensor.to(torch.uint8)
+
+        num_diff_pixels = (out_tensor != out_pil_tensor).sum().item() / 3.0
+        ratio_diff_pixels = num_diff_pixels / out_tensor.shape[-1] / out_tensor.shape[-2]
+        # Tolerance : less than 3% of different pixels
+        assert ratio_diff_pixels < 0.03, "{}: {}\n{} vs \n{}".format(
+            angle, ratio_diff_pixels, out_tensor[0, :7, :7], out_pil_tensor[0, :7, :7]
+        )
+
+    @pytest.mark.parametrize('device', cpu_and_gpu())
+    @pytest.mark.parametrize('height, width', [(26, 26), (32, 26)])
+    @pytest.mark.parametrize('dt', [None, torch.float32, torch.float64, torch.float16])
+    @pytest.mark.parametrize('t', [[10, 12], (-12, -13)])
+    @pytest.mark.parametrize('scripted_affine', [True, False])
+    def test_translations(self, device, height, width, dt, t, scripted_affine):
+        # 3) Test translation
+        tensor, pil_img = _create_data(height, width, device=device)
+
+        if dt == torch.float16 and device == "cpu":
+            # skip float16 on CPU case
+            return
+
+        if dt is not None:
+            tensor = tensor.to(dtype=dt)
+
+        out_pil_img = F.affine(pil_img, angle=0, translate=t, scale=1.0, shear=[0.0, 0.0], interpolation=NEAREST)
+
+        if scripted_affine:
+            fn = torch.jit.script(F.affine)
+        else:
+            fn = F.affine
+
+        out_tensor = fn(tensor, angle=0, translate=t, scale=1.0, shear=[0.0, 0.0], interpolation=NEAREST)
+
+        if out_tensor.dtype != torch.uint8:
+            out_tensor = out_tensor.to(torch.uint8)
+
+        _assert_equal_tensor_to_pil(out_tensor, out_pil_img)
+
+    @pytest.mark.parametrize('device', cpu_and_gpu())
+    @pytest.mark.parametrize('height, width', [(26, 26), (32, 26)])
+    @pytest.mark.parametrize('dt', [None, torch.float32, torch.float64, torch.float16])
+    @pytest.mark.parametrize('a, t, s, sh, f', [
+        (45.5, [5, 6], 1.0, [0.0, 0.0], None),
+        (33, (5, -4), 1.0, [0.0, 0.0], [0, 0, 0]),
+        (45, [-5, 4], 1.2, [0.0, 0.0], (1, 2, 3)),
+        (33, (-4, -8), 2.0, [0.0, 0.0], [255, 255, 255]),
+        (85, (10, -10), 0.7, [0.0, 0.0], [1, ]),
+        (0, [0, 0], 1.0, [35.0, ], (2.0, )),
+        (-25, [0, 0], 1.2, [0.0, 15.0], None),
+        (-45, [-10, 0], 0.7, [2.0, 5.0], None),
+        (-45, [-10, -10], 1.2, [4.0, 5.0], None),
+        (-90, [0, 0], 1.0, [0.0, 0.0], None),
+    ])
+    @pytest.mark.parametrize('i', [NEAREST, ])
```

Comment:
I know it was already there but since there's only one element here, we can remote this parametrization.

Reply:
So, I should just mention `NEAREST` in the function argument directly right ?

Reply:
yes :)